transformation/src/main/scala/org/broadinstitute/monster/clinvar/Args.scala

```diff
@@ -1,10 +1,10 @@
 package org.broadinstitute.monster.clinvar
 
 import caseapp.{AppName, AppVersion, HelpMessage, ProgName}
-import org.broadinstitute.monster.ClinvarIngestBuildInfo
+import org.broadinstitute.monster.ClinvarTransformationPipelineBuildInfo
 
 @AppName("ClinVar transformation pipeline")
-@AppVersion(ClinvarIngestBuildInfo.version)
+@AppVersion(ClinvarTransformationPipelineBuildInfo.version)
```

Comment:
Required because the project name changed, and we use that name to generate this object.